lib/workflowmgr/lsfbatchsystem.rb

```diff
@@ -559,7 +559,6 @@ def run_bhist_bjobs(nacctfiles=1,bjobs=true)
         completed_jobs=""
         errors=""
         exit_status=0
-        timeout=nacctfiles==1 ? 30 : 90
```

Comment:
Removing this will cause problems on WCOSS where long bhist calls often take as long as 70 seconds.

Reply:
It isn't used anymore.  It has been replaced by the configurable timeouts.  It is dead code.

Reply:
It should have been removed in a previous PR, but I missed it.